src/parser/message.rs

```diff
@@ -57,35 +59,40 @@ pub fn message_set<'a>(input: &'a [u8], size: i32) -> IResult<&'a [u8], MessageS
     if size >= 0 {
       take!(i, size as usize)
     } else {
-      Error(Code(ErrorKind::Custom(InputError::InvalidMessageSetSize.to_int())))
+      Error(ErrorKind::Custom(InputError::InvalidMessageSetSize.to_int()))
     }
   };
 
-  flat_map!(input, ms_bytes, |msb| {
-    chain!(
-      msb,
-      messages: message_set_messages,
-      || {
-        messages
-      })
-  })
+  flat_map!(input, ms_bytes, |msb| toto(msb))
```

Comment:
why not write `toto` instead of `|msb| toto(msb)`?

Reply:
Removed because no we use messages_set_messages directly.